opentelemetry-sdk/tests/trace/test_trace.py

```diff
@@ -903,6 +903,38 @@ def test_span_override_start_and_end_time(self):
         span.end(end_time)
         self.assertEqual(end_time, span.end_time)
 
+    def test_span_set_status(self):
+
+        span1 = self.tracer.start_span("span1")
+        span1.set_status(Status(status_code=StatusCode.ERROR))
+        self.assertEqual(span1.status.status_code, StatusCode.ERROR)
+        self.assertEqual(span1.status.description, None)
+
+        span2 = self.tracer.start_span("span2")
+        span2.set_status(
+            Status(status_code=StatusCode.ERROR, description="desc")
+        )
+        self.assertEqual(span2.status.status_code, StatusCode.ERROR)
+        self.assertEqual(span2.status.description, "desc")
+
+        span3 = self.tracer.start_span("span3")
+        span3.set_status(StatusCode.ERROR)
+        self.assertEqual(span3.status.status_code, StatusCode.ERROR)
+        self.assertEqual(span3.status.description, None)
+
+        span4 = self.tracer.start_span("span4")
+        span4.set_status(StatusCode.ERROR, "span4 desc")
+        self.assertEqual(span4.status.status_code, StatusCode.ERROR)
+        self.assertEqual(span4.status.description, "span4 desc")
+
+        span5 = self.tracer.start_span("span5")
+        span5.set_status(
+            Status(status_code=StatusCode.ERROR, description="desc"),
+            description="ignored",
```

Comment:
This particular situation should raise a warning to inform the user that `description="ignored"` was in fact ignored.

Reply:
Fair point, What message text would you suggest?

Reply:
Something like `f""Description {description} ignored when setting status"`.